### PR TITLE
Add window/door sensor to integration

### PR DESCRIPTION
**Pull request by kingsleyadam**
This PR adds the Window/Door sensor to the integration.

### CODE REVIEW

**Comment by kingsleyadam** [comment]
@derjoerg, I'll need your help testing this one as well.

**Comment by nrenner73** [comment]
Should this also work for the wireless window position sensors like 6222/1 AP-66-WL?

**Comment by kingsleyadam** [comment]
> Should this also work for the wireless window position sensors like 6222/1 AP-66-WL?

Do you have the device configuration for this? I'd be able to double check based on that.

**Comment by derjoerg** [comment]
I got the following error:
```
Logger: homeassistant.components.binary_sensor
Source: helpers/entity_platform.py:369
integration: Binary sensor (documentation, issues)
First occurred: 11:05:04 PM (1 occurrences)
Last logged: 11:05:04 PM

Error while setting up abbfreeathome_ci platform for binary_sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 369, in _async_setup_platform
    await asyncio.gather(*pending)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 628, in async_add_entities
    for entity in new_entities:
  File "/config/custom_components/abbfreeathome_ci/binary_sensor.py", line 80, in <genexpr>
    FreeAtHomeBinarySensorEntity(
  File "/config/custom_components/abbfreeathome_ci/binary_sensor.py", line 115, in __init__
    entity_description_kwargs.pop("translation_key")
KeyError: 'translation_key'
```

File: custom_components/abbfreeathome_ci/binary_sensor.py [hunk 1]
```diff
@@ -97,6 +109,11 @@ def __init__(
         self._value_attribute = value_attribute
         self._sysap_serial_number = sysap_serial_number
 
+        # If the channel name is different from the device name, it's likely
+        # a dedicated sensor with it's own naming convention. Use it's channel name instead.
+        if device.channel_name != device.device_name:
+            entity_description_kwargs.pop("translation_key")
```
Comment (derjoerg) [inline custom_components/abbfreeathome_ci/binary_sensor.py:115]: After I added the additional if
```
if 'translation_key' in entity_description_kwargs.keys():
    entity_description_kwargs.pop("translation_key")
```
it is working like a charme

Comment (kingsleyadam) [reply custom_components/abbfreeathome_ci/binary_sensor.py:115]: Ok, a bit odd. All should have the key. I can add the check just to be sure though.

Comment (derjoerg) [reply custom_components/abbfreeathome_ci/binary_sensor.py:115]: I also don't understand it after checking your code ...

**Comment by kingsleyadam** [comment]
Addresses https://github.com/kingsleyadam/local-abbfreeathome-hass/issues/41

**Pull request merged**